scripts/cmscp.py

```diff
@@ -1134,6 +1135,7 @@ def update_exit_info(exit_info, exit_code, exit_msg = None, force = False):
                              'CRAB_StageoutPolicy', \
                              'CRAB_Destination', \
                              'CRAB_Dest', \
+                             'CRAB_Final_Dest', \
```

Comment:
Consistent naming is the only real value. So we need to define things
in one place only. But there has to be a place where definition is clarified,
there is no "correct" name, words like "final" will always mean different
things to different people in different contexts.

On 11/26/2015 05:20 PM, Andres Tanasijczuk wrote:
> Since we use in the CRAB configuration the parameter |outLFNDirBase| maybe here we can use
> |CRAB_OutLFNDir| and |CRAB_OutTempLFNDir| (or |CRAB_TempOutLFNDir|, not sure which one is more correct)?
>